### ISSUE TITLE
Apply only locally works incorrectly

### ISSUE THREAD

**Issue by PanJohnny** [PanJohnny/NoArrowsInYou#1]
It does the opposite. Easy fix in the main class.

**Comment by PanJohnny** [comment]
There is not ! in the if
```java
 ​    ​public​ ​static​ ​boolean​ ​isLocalPlayer​(​Entity​ ​entity​) { 
 ​        ​if​(NAIY​.​getConfig​().​onlyLocal​) 
 ​            ​return​ ​true​; 
 ​        ​return​ ​MinecraftClient​.​getInstance​().​player​ != ​null​ && ​MinecraftClient​.​getInstance​().​player​.​getUuid​().​equals​(​entity​.​getUuid​()); 
 ​    }
```

**Comment by PanJohnny** [comment]
Fix in [a2522b49e9106e2f8e32752fd2253867833ec478](https://github.com/PanJohnny/NoArrowsInYou/commit/a2522b49e9106e2f8e32752fd2253867833ec478)
